src/interpreter.py

```diff
@@ -132,7 +137,19 @@ def visit_variable_expression(self, variable_expression) -> RuntimeValue:
         return self.environment.get(variable_expression.name)
 
     def visit_call_expression(self, call_expression) -> RuntimeValue:
-        pass
+        callee = self.evaluate_expression(call_expression.callee)
+        arguments = [self.evaluate_expression(arg) for arg in call_expression.arguments]
+
+        if not isinstance(callee, Function):
```

Comment:
Can we please discuss during the meeting? I follow what we're trying to do here, but it's a bit unexpected that our runtime relies on isinstance. Perhaps better to just check callee.data_type?

Reply:
Will write is_function method in RuntimeValue